components/Base/ImageZoom/index.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div
+    class="relative bg-white w-full h-full p-4 flex flex-col justify-center items-center dark:bg-black gap-[22px] mt-auto"
+  >
+    <swiper
+      class="absolute transition-transform duration-300 transform h-[450px] align-middle"
+      :slides-per-view="2"
+      :space-between="10"
+      :centered-slides="true"
+      observer
+      observe-parents
+      @swiper="getSwipperRefHandle"
+      @slideChange="updateZoomLevel"
+    >
+      <swiper-slide v-for="(item, index) in loopingDummy" :key="index">
+        <img
+          :src="src"
+          :style="computedZoomStyles(index)"
+          class="h-full w-auto"
+        >
```

Comment:
### Minor

Hello @rayfajars, I think we can `alt` attribute in `img` tag. The `alt` attribute is used to specify the alternate text for an image. It is useful when the image is not displayed. It is used to give alternative information for an image.

Reply:
hallo mr @doohanas ,

is it okay if the alt is the same?

remember because the image is obtained directly via the API

Reply:
I see, we can bind the `alt` attrs, For example :
```js
 <img
          :src="src"
          :alt="`gambar pencarian - ${index}`"
/>
```
or, if you have `title` key from `loopingDummy` you can bind like this: 
```js
 <img
          :src="src"
          :alt="title"
/>
```

Reply:
ah i see,

i fix here [here](https://github.com/jabardigitalservice/super-app-web/pull/63/commits/f6ca27f0af8fc28a25b26db5b9ff1b75227d90a0)